server/src/main/java/org/elasticsearch/Build.java

```diff
@@ -22,50 +22,14 @@
 /**
  * Information about a build of Elasticsearch.
  */
-public record Build(Flavor flavor, Type type, String hash, String date, boolean isSnapshot, String version) {
+public record Build(Type type, String hash, String date, boolean isSnapshot, String version) {
 
     /**
      * The current build of Elasticsearch. Filled with information scanned at
      * startup from the jar.
      */
     public static final Build CURRENT;
 
-    public enum Flavor {
-
-        DEFAULT("default"),
-        OSS("oss"),
-        UNKNOWN("unknown");
-
-        final String displayName;
-
-        Flavor(final String displayName) {
-            this.displayName = displayName;
-        }
-
-        public String displayName() {
-            return displayName;
-        }
-
-        public static Flavor fromDisplayName(final String displayName, final boolean strict) {
-            switch (displayName) {
-                case "default":
-                    return Flavor.DEFAULT;
-                case "oss":
-                    return Flavor.OSS;
-                case "unknown":
-                    return Flavor.UNKNOWN;
```

Comment:
What do we use for the integ-test distribution? Was this "unknown" before?

Reply:
Yes it would have been unknown, though I don't think it matters, since nothing will be checking it now.